easybuild/tools/toolchain/toolchain.py

```diff
@@ -727,6 +729,15 @@ def prepare_compiler_cache(self, cache_tool):
         self.cached_compilers.update(compilers)
         self.log.debug("Cached compilers (after preparing for %s): %s", cache_tool, self.cached_compilers)
 
+    @staticmethod
+    def is_rpath_wrapper(path):
+        """
+        Check whether command at specified location already is an RPATH wrapper script rather than the actual command
+        """
+        in_rpath_wrappers_dir = os.path.basename(os.path.dirname(path)) == RPATH_WRAPPERS_SUBDIR
+        calls_rpath_args = 'rpath_args.py $CMD' in read_file(path)
```

Comment:
I don't necessarily want the check to see whether the command I'm wrapping is a wrapper script already to be strict. On the contrary, I'd rather do a less strict non-hash check (like I'm doing now), which limits the chance of running into a false negative and wrapping an existing wrapper...